spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -274,6 +274,10 @@ def create_application!(name: nil, primary_language: nil, version: nil, sku: nil
       r = request(:get, "ra/apps/create/v2/?platformString=#{platform}")
       data = parse_response(r, 'data')
 
+      # check if the bundle id exists in the list of bundle ids
+      available_bundle_ids = data['bundleIds']
+      raise 'requested bundle id not available' unless available_bundle_ids.keys.include? bundle_id
```

Comment:
Please add `( )` around `bundle_id` for method calls. Also, can we change the error message to `Bundle identifier provided is already taken and not available any more. Please choose a different one`